scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,22 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    local INSTANCE_TEMPLATE_REQ=$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
```

Comment:
What if user does not specify SOURCE_INSTANCE_TEMPLATE? For example, user just want to create debian VM with the default setup.

`INSTANCE_TEMPLATE_REQ` is not accurate, this is more about getting SOURCE_INSTANCE_TEMPLATE, it is not relevant to any requirement.

Reply:
Just use single local variable should be enough, as you are only verifying the image path in this function. Please capitalize the local variables as well to be consistent with other local variables.